sway/tree/view.c

```diff
@@ -164,7 +164,7 @@ void view_set_fullscreen_raw(struct sway_view *view, bool fullscreen) {
 		wl_list_for_each(seat, &input_manager->seats, link) {
 			focus = seat_get_focus(seat);
 			focus_ws = focus;
-			if (focus_ws->type != C_WORKSPACE) {
+			if (focus && focus_ws->type != C_WORKSPACE) {
```

Comment:
If `focus` is `NULL`, `focus_ws` will also be `NULL`. Then the if-statement on line 171 will always succeed, and you'll pass `NULL` to `seat_set_focus`. It looks like `seat_set_focus` will handle `NULL` gracefully, but I'd rather wrap lines 166-173 in an `if (focus)` check to make it obvious to the reader that this only applies if the seat has focus somewhere.